netebpfext/net_ebpf_ext.c

```diff
@@ -276,7 +348,7 @@ net_ebpf_ext_register_callouts(_Inout_ void* device_object)
 
     RtlZeroMemory(&ebpf_hook_sub_layer, sizeof(FWPM_SUBLAYER));
 
-    ebpf_hook_sub_layer.subLayerKey = EBPF_HOOK_SUBLAYER;
+    ebpf_hook_sub_layer.subLayerKey = EBPF_SUBLAYER;
     ebpf_hook_sub_layer.displayData.name = L"EBPF hook Sub-Layer";
```

Comment:
```suggestion
    ebpf_hook_sub_layer.displayData.name = L"eBPF Sub-Layer";
```
consistency with change to previous line